cylc/flow/network/server.py

```diff
@@ -299,7 +298,6 @@ def receiver(self, message) -> 'ResponseDict':
         try:
             method = getattr(self, message['command'])
             args = message['args']
-            args.update({'user': message['user']})
```

Comment:
I have checked inter-version compatibility by running a workflow in this PR's version of Cylc and the UIS running at Cylc 8.4.0, and testing mutations from the UI still work. And vice versa.